src/commands/node/uninstall.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { flags as flagParser } from '@oclif/command';
+import * as fsExtra from 'fs-extra';
+import Listr from 'listr';
+import BaseCommand from '../../base';
+import { NETWORK } from '../../utils/constants';
+import { flags as commonFlags } from '../../utils/flags';
+import { defaultInstallationPath } from '../../utils/node/config';
+import {
+	describeApplication,
+	Pm2Env,
+	unRegisterApplication,
+} from '../../utils/node/pm2';
+import StopCommand from './stop';
+
+interface Flags {
+	readonly installationPath: string;
+	readonly name: string;
+	readonly network: NETWORK;
+}
+
+export default class UnInstallCommand extends BaseCommand {
+	static description = 'UnInstall Lisk Core';
+
+	static examples = [
+		'node:uninstall --name=mainnet_1.6',
+		'node:uninstall --network=testnet --name=testnet_1.6',
+	];
+
+	static flags = {
+		...BaseCommand.flags,
+		network: flagParser.string({
```

Comment:
also, network can be obtained from `pm2` setting at this point

Reply:
yes we can, but the thing is, you will have many instances, so its better that users gives which network and which instance, for example
```
mainnet
    1.5
    1.6
testnet
    1.5
    1.6
```

now if user uninstall he should know which one to `--network=mainnet --name=1.5`

Reply:
but what if
```
mainnet
1.5
testnet
1.5
1.6
```
with the command `--network=mainnet --name=1.6` whats the correct behavior ?